etcdctl/ctlv3/command/del_command.go

```diff
@@ -70,6 +74,10 @@ func getDelOp(args []string) (string, []clientv3.OpOption) {
 			cobrautl.ExitWithError(cobrautl.ExitBadArgs, fmt.Errorf("too many arguments, only accept one argument when `--prefix` or `--from-key` is set"))
 		}
 		opts = append(opts, clientv3.WithRange(args[1]))
+		if !delRange {
+			fmt.Fprintln(os.Stderr, "Warning: Please add flag `--range` to make it explicit and avoid 2s delay.")
```

Comment:
I don't think we should warn about the time delay :P 
```suggestion
			fmt.Fprintf(os.Stderr, "Warning: Keys between %q and %q will be deleted. Please interrupt the command within 2 seconds to cancel. You can provide `--range` flag to avoid the delay.\n")
```